build/index.js

```diff
@@ -266,12 +267,40 @@ function logSuccess(command, timeLapsed) {
     logMain(message);
 }
 
+function checkVersions(metaManifest) {
+    const thisNPM = process.platform === "win32" ? "npm.cmd" : "npm";
+    let exitCode = 0;
+    const nodeMajorVersion = parseInt(process.version.slice(1, 3), 10);
+    if (nodeMajorVersion < metaManifest.requiredNodeMajorVersion) {
+        console.error(
+            `⚠️ Node.js v${metaManifest.requiredNodeMajorVersion} or later ` +
+                `required, you are on ${process.version}`
+        );
```

Comment:
Nit: I would find it more elegant to `throw` this error message (just as a string), and to `catch` it and log it where you call `checkVersions`. I guess one benefit of this implementation is that we would actually always check _both_ the Node version and the NPM version, but I guess if the user has the wrong Node version they'll end up with a different NPM version after upgrading that anyway.

Reply:
Well, was just a nit, so I'll live :D